### DISCUSSION TITLE
How does the interactive folder structure work?

### DISCUSSION THREAD

**Discussion by ianzone** [biomejs/website#1561] [Q&A] [answered]
I found [this component](https://biomejs.dev/guides/configure-biome/#configuration-file-resolution) interesting, is it done by a lib? 
<img width="281" alt="image" src="https://github.com/user-attachments/assets/5afe252c-c0a0-49f8-8fb6-6f1dbf50f13a" />

**Comment by ematipico** [comment, marked as answer]
It's provided by starlight!

https://starlight.astro.build/components/file-tree/

**Reply by ianzone** [reply]
cool👍
